README.md

```diff
@@ -838,4 +838,6 @@ The relationship between these is:
 
 ## JetStream
 
-JetStream is the NATS persistence engine providing streaming, message, and worker queues with At-Least-Once semantics. [Support for JetStream is built-in](jetstream.md).
+JetStream is the NATS persistence engine providing streaming, message, and
```

Comment:
I have seen this change in an older PR, no? Could it be that your branch was out-of-date? Maybe a rebase would help?

Reply:
the nats.js has the same comment - the current version of deno reformatted that file-